SDK v2 Examples/Mall Sample App/Mall Sample App/TextDirectionsViewController.swift

```diff
@@ -31,7 +31,8 @@ class TextDirectionsViewController: UITableViewController {
         
         let instruction = directions.instructions[indexPath.row]
         if ( indexPath.row > 0 ) {
-            cell.detailTextLabel?.text = "In about " + String(format: "%.1f", instruction.distance) + "m"
+        
+            cell.detailTextLabel?.text = "In about " + String(format: "%.1f", instruction.distance ?? 0.0) + "m"
```

Comment:
I actually think we should not show the distance at all if the distance is nil. We should not include this line in those cases